source/jormungandr/tests/ptref_tests.py

```diff
@@ -119,6 +119,28 @@ def test_line(self):
         assert physical_modes[0]['id'] == 'physical_mode:Car'
         assert physical_modes[0]['name'] == 'name physical_mode:Car'
 
+        group = get_not_null(l, 'line_groups')
+        assert len(group) == 1
+        assert group[0]['name'] == 'A group'
+        assert group[0]['id'] == 'group:A'
+
+    def test_line_groups(self):
+        """test line group formating"""
+        response = self.query_region("v1/line_groups")
+
+        line_groups = get_not_null(response, 'line_groups')
+
+        assert len(line_groups) == 1
+
+        lg = line_groups[0]
+
+        is_valid_line_group(lg, depth_check=1)
+
+        com = get_not_null(lg, 'comments')
+        assert len(com) == 1
+        assert com[0]['type'] == 'standard'
+        assert com[0]['value'] == "I'm a happy comment"
```

Comment:
it's really great that you added some jormungandr tests! Since we decided to do some non conventional stuffs with the depth I think it would be great to add some more tests:

* one test per depth there just to be sure (after the call, a call to is_valid_line_group should be enough)
* some tests to check that the ptref stuff work well with the line_group: 
   * `self.query_region("v1/line_groups/{line group id}/lines")`
   * and maybe `self.query_region("v1/routes/{one route id}/line_groups")` to check if you can get all line_groups for a route